CONTRIBUTING.md

```diff
@@ -196,11 +196,10 @@ There are a few things to think about when doing a release:
 - Create a PR containing the updated changelog and `pubspec.yaml` files.
 
 
-[GitHub issue]: https://github.com/flame-engine/flame/issues/new
-[GitHub issues]: https://github.com/flame-engine/flame/issues/new
+[GitHub issue]: https://github.com/flame-engine/flame/issues
```

Comment:
nit: I would still call this `GitHub issues`

Reply:
done